lib/membrane_opus/description.ex

```diff
@@ -1,37 +1,46 @@
 defmodule Membrane.Opus.Description do
   @moduledoc """
-  This module implements struct describing a single Opus-encoded audio frame.
+  This module implements struct describing an Opus-encoded audio stream.
+
+  Based on [RFC 6716](https://tools.ietf.org/html/rfc6716).
   """
 
-  # Opus frame duration type.
-  #
-  # 2 means 2.5ms, other values just map into duration in miliseconds.
-  @type frame_duration_t :: 2 | 5 | 10 | 20 | 40 | 60
+  # TODO Support combined multiple frames (up to 120ms).
+  @typedoc """
+  Opus frame duration in milliseconds.
+  """
+  @type frame_duration_t :: 2_5 | 5 | 10 | 20 | 40 | 60
```

Comment:
I was sure you meant an atom `:2_5` when we spoke about it. `2_5` is `25` so it's even worse than `2`

Reply:
well `:2_5` is invalid syntax, I would have to use `:'2_5'` :/

Reply:
I propose to retract to `float` and leave allowed values to be specified by docs; just as it's done for bitrate

Reply:
The additional advantage is that it would play nicely with combined frames (up to 120ms, but not quantized)

Reply:
It would be better to use Membrane.Time instead of floats, but I've investigated the RFC a bit further and it turns out the frame duration can be different for each Opus packet! Therefore frame_duration does not belong here, should be buffer metadata. Problem solved :smile: